src/store/CurrentAccount/getters.ts

```diff
@@ -0,0 +1,16 @@
+import { AccountTypes } from '@/enums'
+import { CurrentAccountInterface } from '@/interfaces'
+
+
+export default {
+
+  /** Whether the current account is a premium account. */
+  isPremiumAccount (state: CurrentAccountInterface): boolean {
+    return (state.accountType === AccountTypes.PREMIUM)
+  },
+
+  /** Whether the user is ServiceBC Staff (which is not the same as Staff). */
+  isSbcStaff (state: CurrentAccountInterface): boolean {
+    return (state.accountType === AccountTypes.SBC_STAFF)
+  }
+}
```

Comment:
Any idea if we can add `isStaff` here (instead of check the auth roles in the KC token as we do now)? Maybe double-check with Travis. Maybe we never need to parse the KC token.

Reply:
No, unfortunately not, the Keycloak resource is separate from the resource that determine Premium vs SBC_STAFF accounts, but I do think we should create a store for the Keycloak resource.

Reply:
Travis confirmed that `CURRENT_ACCOUNT.accountType` (eg, "STAFF") is not in sync with the Keycloak (jwt) roles array?

No matter if not. It would have been a nice cleanup.

Reply:
@severinbeauvais No, I traced the code back and can see it's pulling from a different endpoint and getting back a different response. I had a 💡 moment when I realized that I after I've logged into via Keycloak, I can change my CurrentAccount -- one login has many accounts.  Here's the response from the current account endpoint:

![Screenshot (25)](https://user-images.githubusercontent.com/25233684/217964601-c096a32e-29e1-4dd8-a0c7-e52c5598fab4.png)